### PR TITLE
Github repo spring cleaning

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
Hoping to address some pain points I and new contributors experienced with getting started and testing. I've also updated the screenshot and some structure.  

Preview: https://github.com/OskarDamkjaer/neo4j-browser/tree/repo_cleanup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,8 @@
     "starts": "cross-env NODE_ENV=development webpack-dev-server --https --colors --config ./build_scripts/webpack.config.js",
     "start-prod": "cross-env NODE_ENV=production webpack-dev-server --colors --config ./build_scripts/webpack.config.js",
     "test": "npm run lint && npm run jest",
+    "test-unit": "test",
+    "test-e2e": "npx concurrently --kill-others -n 'neo4j,cypress,browser' 'docker run --env NEO4J_AUTH=neo4j/newpassword -p7687:7687 neo4j:4.2.2' 'yarn wait-on-neo4j && yarn wait-on-dev && yarn e2e-local --env server=4.2' 'yarn && yarn start'",
```
Comment (HerrEmil) [inline package.json:41]: This is failing on my machine, wait-on times out. Neo4j and yarn start seem to start as expected and on expected ports, but then nothing happens until wait-on times out.

Comment (HerrEmil) [reply package.json:41]: port 7474 should be forwarded

Comment (OskarDamkjaer) [reply package.json]: Thanks!

File: README.md [hunk 2]
```diff
@@ -1,62 +1,66 @@
 # Neo4j Browser
 
-Neo4j Browser is the general purpose user interface for working with Neo4j. Query, visualize, administrate and monitor the database
-with modern and easy-to-use tools.
+Neo4j Browser is the general purpose graphical user interface for Neo4j.
 
 ![neo4j browser screenshot](./.github/neo4j-browser-screenshot.png)
```
Comment (jharris4) [inline README.md:5]: Reminder: need a new screenshot for the new branding...

File: README.md [hunk 3]
```diff
@@ -1,62 +1,66 @@
 # Neo4j Browser
 
-Neo4j Browser is the general purpose user interface for working with Neo4j. Query, visualize, administrate and monitor the database
-with modern and easy-to-use tools.
```
Comment (jharris4) [inline README.md:4]: Not sure we need to remove all these terms, especially maybe we should keep the query & visualize parts

Comment (OskarDamkjaer) [reply README.md:4]: Done.

**Comment by oskarhane** [comment]
This one needs a rebase @OskarDamkjaer

**Comment by OskarDamkjaer** [comment]
> This one needs a rebase @OskarDamkjaer

Done!

File: README.md [hunk 4]
```diff
@@ -1,62 +1,66 @@
...
-Here are the available options / env variables:
+To avoid opening the Cypress test runner and just run the tests in the terminal, remove the "-open" suffix from the previous two commands (so `yarn e2e` and `yarn e2e-local` respectively).
 
+So to run tests on your existing 4.2 database with the password "hunter2" without opening the Cypress visual test runner:
+`yarn e2e-local --env browser-password=hunter2,server=4.2`
+
+All the available options for `--env` are:
 ```
-server=3.2|3.3|3.4|3.5|4.0|4.1 (default 3.5)
+server=3.5|4.0|4.1|4.2 (default 3.5)
```
Comment (oskarhane) [inline README.md:51]: Can we change the default to be 4.3 (and also add 4.3 to the list)?

Comment (OskarDamkjaer) [reply README.md]: Updated, let's see if the tests still pass

**Review by oskarhane — APPROVED** [review]
LGTM 🏪

**Pull request merged**